release.yaml

```diff
@@ -1,5 +1,5 @@
 kuadrant-operator:
-  version: "1.2.0-rc1"
+  version: "1.2.0-rc2"
 olm:
   channels:
     - "alpha"
```

Comment:
we can leave this as alpha. The workflow generating the catalog has `stable` hardcoded https://github.com/Kuadrant/kuadrant-operator/blob/main/.github/workflows/build-images-for-tag-release.yaml#L173